### ISSUE TITLE
Gem not available from a different RVM environment

### ISSUE THREAD

**Issue by navaneeth** [getgauge/gauge-ruby#2]
I did `gauge --init ruby` from a directory which uses a `.rvmrc` file and a different gemset. `.rvmrc` has the following:

```
rvm use ruby-2.1.3@project-name --create
```
Now when I did `init`, gauge installed ruby into the above gemset. Now when I move to a different directory and initialize a new gauge project, and run `gauge specs`, it fails with error:

```
/Users/navaneethkn/.rvm/rubies/ruby-2.1.3/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- gauge-runtime (LoadError)
	from /Users/navaneethkn/.rvm/rubies/ruby-2.1.3/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from -e:1:in `<main>'
Failed to start Gauge Ruby runner. exit status 1
Runner exited with error: exit status 1
```

I think the way ruby runner is packaged is wrong. It can't be a gem. If it has to be a gem, it should get installed to the global gem path where the you may need permissions. Ideally, it should search for the files in the plugin installed path and modify the load path to include this so that it always works irrespective of where it is installed

**Comment by sriv** [comment]
This should be fixed now, with the introduction of `gauge-ruby` in rubygems.org, the install process for `gauge-ruby` has changed slightly.

Instead of bundling the gem file and doing a `user-install` of the gem, the plugin copies a `Gemfile` to the target project (appends if one already exists) and aggs `gauge-ruby` as a `development` and `test` dependency. Then the plugin invokes `bundle install`.

**Comment by navaneeth** [comment]
Nice. Is this released? When is 0.0.4 coming out?

**Comment by sriv** [comment]
There is still [one issue](https://github.com/getgauge/gauge-ruby/issues/6) pending, which I am looking at. That would make it ready for [milestone 0.0.4](https://github.com/getgauge/gauge-ruby/milestones/0.0.4). Hopefully that should not take much time :)
